bin/json-to-envvars

```diff
@@ -16,9 +17,13 @@
 #   ENV2<<__EOF__
 #   DEF
 #   __EOF__
+#   ENV3<<__EOF__
+#
+#   __EOF__
```

Comment:
Ah, I should probably add more detail to the commit message for this change. 

I made this change because a test seasonal-flu build failed when it passed `NEXTSTRAIN_DOCKER_IMAGE: null` to the Nextstrain CLI and the AWS Batch job tried to pull the `null` image. With this change, the Nextstrain CLI is able to see this is environment variable as a falsy value and pulled the latest image.

Reply:
Ah, got it!